pkg/apis/provisioning/v1alpha5/constraints.go

```diff
@@ -54,7 +55,11 @@ func (c *Constraints) ValidatePod(pod *v1.Pod) error {
 	// Test if pod requirements are valid
 	requirements := NewPodRequirements(pod)
 	if errs := requirements.Validate(); errs != nil {
-		return fmt.Errorf("pod requirements not feasible, %v", errs)
+		var multiErrors error
+		for _, err := range errs {
+			multiErrors = multierr.Append(multiErrors, fmt.Errorf(err))
+		}
+		return fmt.Errorf("pod requirements not feasible, %w", multiErrors)
```

Comment:
This reads poorly. `are not feasible` is more correct, but that's pretty verbose. I might just say "invalid requirements, %w".

With the caller of this, it would read as, `Unable to schedule pod foo, invalid requirements, reasons...`

Reply:
IMO we need to give users a hint on what's failing. (In this case, the constraints from the pod.Spec is illy constructed.)
This is mainly catching the provisioner matching errors. So it would read as.
`Could not schedule pod, matched 0/3 provisioners, tried provisioner/default: pod requirements not feasible, some reason`

If we just say "invalid requirements" in the error message, we won't be able to tell if it is caused by provisioner requirements or pod requirements.

Reply:
Oh I see -- I think this problem is caused by not all consumers logging in an intelligible way. As is, your change would repeat the word pod: `unable to schedule pod foo, pod requirements...`. 

In your error `Could not schedule pod`, I think it's already pretty well contextualized given this prefix.